libs/frontend/domain/builder/src/sections/content/Builder.tsx

```
@@ -62,6 +67,13 @@ export const Builder = observer<BuilderProps>(
       setCurrentBuilderWidth,
     })
 
+    // by our current IElement, root can be undefined
+    // useDroppable does not accept an undefined id
+    // since this is the root element, id should be defined
+    const { setNodeRef, isOver, over } = useDroppable({
+      id: elementTree.root?.id as string,
```

should throw error before if undefined?